src/views/preview/preview.jsx

```diff
@@ -312,6 +324,29 @@ class Preview extends React.Component {
             title: title
         });
     }
+    handleUpdateProjectId (projectId, callback) {
+        // NOTE: this needs more work
+        // NOTE: need to load everything for new project that is loaded in componentDidUpdate
+        this.setState({projectId: projectId}, () => {
+            const parts = window.location.pathname.toLowerCase()
+                .split('/')
+                .filter(Boolean);
+            let newUrl;
+            if (projectId === 0) {
+                newUrl = `/${parts[0]}/editor`;
+                this.props.resetProject();
+            } else {
+                newUrl = `/${parts[0]}/${projectId}/editor`;
+                this.fetchCommunityData();
+            }
+            history.pushState(
+                `project ${projectId}`,
+                `project ${projectId}`,
```

Comment:
Might be more useful to us in the future if this state was something we could use easily, like `{projectId: projectId}`.

Reply:
Done